app/javascript/article-form/articleForm.jsx

```diff
@@ -361,7 +361,9 @@ export default class ArticleForm extends Component {
           submitting={submitting}
         />
 
-        <KeyboardShortcutsHandler togglePreview={this.fetchPreview} />
+        <KeyboardShortcuts shortcuts={{
+          "ctrl+shift+KeyP": e => this.fetchPreview(e)
```

Comment:
I think by prettier's rules in this repo, this line needs `(e)` and a trailing comma